developer-docs-site/docs/concepts/coin-and-token/token_airdrop/Token Airdrop Example c4d35e65f7314699a5335fb96d2d5c1f.md

```diff
@@ -0,0 +1,29 @@
+# Token Airdrop Example
+
+We support different ways of transferring token in our token standard [[link](https://aptos.dev/concepts/coin-and-token/aptos-token#token-transfer)]
+
+With the 2 step token transfer, airdropping NFTs to the a set of address can be very straight-ward. 
```

Comment:
we should avoid unnecessary superlatives ...

Reply:
removed